packages/@okta/vuepress-theme-prose/index.js

```diff
@@ -14,9 +14,9 @@ module.exports = (options, ctx) => {
       $page.fullHeaders = resolveHeaders($page);
 
       // Set to true to enable the redesign in development
-      const redesign = process.env.REDESIGN || false;
+      //const redesign = process.env.REDESIGN || false;
       // Respect the redesign flag unless this is a production build
-      $page.redesign = !ctx.isProd && redesign;
+      $page.redesign = true;//!ctx.isProd && redesign;
```

Comment:
🚨 🚨 🚨  note to remove 🚨 🚨 🚨

Reply:
nit: would add `$page.redesign = true;` after line 19 is good enough without changing anything else?

Reply:
I only put this temporarily for the Netlify deployment but I will revert it before merging